test-suite/output/Qf_end.out

```diff
@@ -1,7 +1,8 @@
 File "./output/Qf_end.v", line 2, characters 4-5:
-Error: Last block to end has name A.
+Error:
+Last block to end has name A.
```

Comment:
Is the linebreak spurious?

Reply:
probably a missing box

Reply:
I looked into it and I could not fix this change.
Since it only affects the case where we pass -test_mode and we have quickfixes, I think it is not a blocker.

Reply:
the code is https://github.com/coq/coq/blob/eb4169c898e02f08b243b03b8d00c8cf7a353828/vernac/topfmt.ml#L142-L147
when there is a quickfix we print `v 0 (hov 0 (tag error (str "Error:") ++ spc() ++ error_message) ++ cut() ++ quickfix_message)`

>I think it is not a blocker.
 
I agree

Reply:
I met once a similar unwanted break but I did not find a solution. So, if ever you find a solution, I'd be interested to know what was the problem.